x-pack/solutions/observability/plugins/streams_app/public/components/schema_editor/schema_editor_table.tsx

```diff
@@ -79,8 +79,10 @@ export function FieldsTable({
       trailingControlColumns={trailingColumns}
       gridStyle={{
         border: 'none',
-        rowHover: 'none',
-        header: 'underline',
+        rowHover: 'highlight',
+        header: 'shade',
+        fontSize: 's',
+        cellPadding: 'm',
```

Comment:
The table seems very compact and difficult to read now, it also differs from the designs on Figma, is this the intended UI?

<img width="1893" alt="Screenshot 2025-02-19 at 17 25 05" src="https://github.com/user-attachments/assets/b634ab4c-f19f-46e3-a3da-89ae5acc7e37" />